ext_mongodb.php

```diff
@@ -21,6 +21,11 @@ public function getMessage() : string
 		return $this->message;
 	}
 
+	public function getInfo() : array
```

Comment:
I just realized we can use nullables on return types (see: [docs](http://docs.hhvm.com/manual/en/hack.nullable.scenarios.php#hack.nullable.scenarios.nonnullableinteraction)), so `?array` should work.